### PR TITLE
Add Material Design resources

### PR DESCRIPTION
**Pull request by FerPerales**

### CODE REVIEW

**Comment by jpguz** [comment]
:+1: LGTM

**Comment by anyelic** [comment]
great

**Comment by jesusharo** [comment]
LGTM

**Comment by jrdnlpz** [comment]
LGTM

File: android_ux_ui.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Android UX/UI 
+
+
+This category gathers all resources related to Android, Material Design and best practices for creating Android apps.
+
+## Material Design
+
+- [What is material design?](http://www.google.com/design/spec/material-design/introduction.html) This spec is a living document that will be updated as we continue to develop the tenets and specifics of material design.
```
Comment (KatyJmz) [inline android_ux_ui.md:8]: que es este texto?  :dancer:

Comment (FerPerales) [inline android_ux_ui.md:8]: Is an abstract of the content of the link :dancer:

Comment (KatyJmz) [inline android_ux_ui.md:8]: ok

**Pull request merged**